suricata/update/main.py

```diff
@@ -1052,7 +1052,11 @@ def _main():
     global_args, rem = global_parser.parse_known_args()
 
     if global_args.version:
-        print("suricata-update version %s (rev: %s)" % (version, revision))
+        if revision != None:
+            revision_string = " (rev: %s)" % (revision)
+        else:
+            revision_string = ""
```

Comment:
Minor: We could do these four lines in a one liner:

`revision_string = " (rev: %s)" % (revision) if revision else ""`